Documentation~/Blender.md

```diff
@@ -69,3 +70,36 @@ This option will only appear if we install [ProBuilder](https://docs.unity3d.com
 > If **Bake Modifiers** option is enabled in the plugin, 
 > a mesh that is sent from Unity back into Blender will have modifiers applied beforehand and have them removed after.
 
+## Material sync mode
+
+The MeshSyncServer in Unity always creates a material with a default shader for the active render pipeline.
+There are multiple options to export materials from blender:
+
+### None
```

Comment:
Please put this None and basic into a table. 
See [this for reference](https://github.com/Unity-Technologies/MeshSyncDCCPlugins/blob/fe9abd792916acaa87049ada5e3114f8995a4d4c/Documentation~/Blender.md?plain=1#L12).

Because the basic option is a bit long, we can write a summary in the table with an anchor to another section.

Reply:
Done